internal/lint/check_file_options_equal.go

```diff
@@ -117,3 +125,9 @@ func packageBasename(pkg string) string {
 	split := strings.Split(pkg, ".")
 	return split[len(split)-1]
 }
+
+func fileBasenameUpperCamelCase(filename string) string {
```

Comment:
We have this same function in `lint` and `format`. We're also doing a bit of adhoc java/go package naming (ie `"foo" +"pb"), thoughts on creating/moving helper functions into the `strings` package?

Reply:
I'm going to do a follow-up that cleans this up, is that OK?

Reply:
yup sounds good!